eldritch/encounters.py

```diff
@@ -144,11 +144,101 @@ def Society4(char):
   cond = events.Conditional(char, check, "successes", {0: events.Nothing(), 2: skill})
   return events.Sequence([check, cond], char)
 
+def Administration1(char):
+  check = events.Check(char, "lore", -1)
+  dollars = events.Gain(char, {"dollars": 5})
+  return events.PassFail(char, check, dollars, events.Nothing())
+def Administration2(char):
+  return events.Gain(char, {"clues": 1})
+def Administration3(char):
+  check =  events.Check(char, "will", -1)
+  retainer = events.StatusChange(char, "retainer")
+  return events.PassFail(char, check, retainer, events.Nothing())
+def Administration4(char):
+  check = events.Check(char, "lore", -2)
+  spell = events.Draw(char, "spells", 1)
+  curse = events.Curse(char)
+  return events.PassFail(char, check, spell, curse)
+def Administration5(char):
+  return events.ForceMovement(char, "Asylum") # TODO: also have an event
```

Comment:
Should be able to just Sequence(ForceMovement, Encounter)

Reply:
bleh, now I need to figure out how to populate the encounter deck for a draw.

Reply:
Actually, it looks like Encounter takes a location not a location name, so there may need to be more discussion about how to do this (or I might be missing something)

Reply:
I'll get back to this later then.